transformation/src/test/scala/org/broadinstitute/monster/hca/HcaPipelineBuilderSpec.scala

```diff
@@ -182,19 +182,19 @@ class HcaPipelineBuilderSpec extends AnyFlatSpec with Matchers with PipelineSpec
   }
 
   it should "correctly generate file ingest requests" in {
-    val exampleHash = "54321zyx"
+    val exampleId = "my-file-id"
     val exampleDescriptor = JsonParser.parseEncodedJson(
       json = s"""
                 | {
                 |    "file_name": "a-directory/sub_directory/file-id_file-version_filename.json",
-                |    "file_id": "my-file-id",
+                |    "file_id": "$exampleId",
                 |    "file_version": "my-file-version",
-                |    "crc32c": "$exampleHash",
+                |    "crc32c": "54321zyx",
```

Comment:
will we continue to get the crc32c in the descriptor json?
was the entire purpose of this for deduping?

Reply:
i don't think i understand the various steps here enough to comment. we'll get the crc32c hash from the HCA regardless, since it's a required field in the file descriptor schema, but i don't know what we're doing with it besides (formerly) using it to dedupe.

Reply:
@quazi-broad none of the source data has any defined purpose for us; we made a previous decision to just use one of the hashes for this step.